README.md

```diff
@@ -97,13 +97,25 @@ Finally, a board can return values based on its state such as `#full?` when enti
 
 ##### `player.rb` - `Player`
 
-The `Player` class is not actually a valid player of Tic Tac Toe but rather a root class that will act as an inheritance point for actual player classes such as `Player::Human` and `Player::Computer`. The `Player` root class will define only the most basic properties of a player, that they have a `token` property that can be set upon initialization.
+The `Player` class is not actually a valid player of Tic Tac Toe but rather a root class that will act as an inheritance point for actual player classes such as `Human < Player` and `Computer < Player`. The `Player` root class will define only the most basic properties of a player, that they have a `token` property that can be set upon initialization.
 
 Every player subclass will implement a `#move` method that represents how that type of player makes a move in Tic Tac Toe.
 
-##### 'players/human.rb' - `Player::Human`
+##### 'players/human.rb' - `Players::Human`
 
-Define a class `Player::Human` that inherits from `Player`. The human player must implement a `#move` method that takes in a `board` argument and allows a human player to enter a move via the CLI. The method should return the value the user enters. Even though the method accepts a `board` argument, it does not need to use it.
+Define a class `Human` that inherits from `Player`. This class should be namespaced inside the module `Players` because the `human.rb` file is inside the `players/` directory. This keeps our code nice and tidy.
+
+The human player must implement a `#move` method that takes in a `board` argument and allows a human player to enter a move via the CLI. The method should return the value the user enters. Even though the method accepts a `board` argument, it does not need to use it.
```

Comment:
Hey @pletcher I think that's a good additional discussion, but how would you feel about raising another issue around that? I just want to make sure we ship small improvements quickly rather than slowing down small improvements by trying to nail everything. Also given the pretty clear naming of this PR I'm feeling like reworking the language around boards is out of scope (although it'd be a great thing to raise as a separate issue and @ mention us if that's cool?